app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,11 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
+        Alerting.AlertAddMsg('bottom', 'warning', 'the key will be exported and should be protected');
```

Comment:
I see in the view there is no alerts.top section.  It is called alerts.role for some reason.  That is a bug